src/main/java/com/intuit/graphql/orchestrator/utils/ValueUtil.java

```diff
@@ -0,0 +1,158 @@
+package com.intuit.graphql.orchestrator.utils;
+
+import static graphql.schema.GraphQLTypeUtil.isList;
+import static graphql.schema.GraphQLTypeUtil.isNonNull;
+
+import graphql.AssertException;
+import graphql.Scalars;
+import graphql.language.ArrayValue;
+import graphql.language.BooleanValue;
+import graphql.language.EnumValue;
+import graphql.language.FloatValue;
+import graphql.language.IntValue;
+import graphql.language.ObjectField;
+import graphql.language.ObjectValue;
+import graphql.language.StringValue;
+import graphql.language.Value;
+import graphql.schema.GraphQLEnumType;
+import graphql.schema.GraphQLInputObjectField;
+import graphql.schema.GraphQLInputObjectType;
+import graphql.schema.GraphQLInputType;
+import graphql.schema.GraphQLList;
+import graphql.schema.GraphQLNonNull;
+import graphql.schema.GraphQLScalarType;
+import graphql.schema.GraphQLType;
+import graphql.schema.PropertyDataFetcherHelper;
+import graphql.util.FpKit;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The code from this is from graphql-java library AstValueHelper
```

Comment:
Have they removed this ?

Reply:
yup.  I was not able to find a replacement.  ValueResolver.java looks like does not have something similar but will revisit that class

Reply:
you may be able to use the Parser.parseValue that is what they changed to

Reply:
I was intrigued so I looked a little into it, the method you may be looking for is ValuesResolver.valueToLiteral() but value would have to be converted to a InputValueWithState

Reply:
just to close the loop on this. we found out the replacement methods for this class and @CNAChino 
should be adding those changes in the next commit

Reply:
Should we remove this class

Reply:
The solution is to use `ValuesResolver.valueToLiteral(inputValueWithState, type)`.   And use `InputValueWithState.newExternalValue()`.  `InputValueWithState.newInternalValue()` will be removed soon.

Reply:
> I was intrigued so I looked a little into it, the method you may be looking for is ValuesResolver.valueToLiteral() but value would have to be converted to a InputValueWithState

👍

Reply:
> Should we remove this class

yes